scripts/test_status.sh

```diff
@@ -32,8 +36,14 @@ echo "Summary: \
    $(find content/en/docs/tasks -name '*.md' -exec grep --quiet '^test: n/a$' {} \; -print | wc -l) (n/a)"
 
 echo ""
+echo "Tested:"
+${green}
+find content/en/docs/tasks -name '*.md' -exec grep --quiet '^test: yes$' {} \; -print
```

Comment:
nit: can `content/en/docs` be declared as a variable ? It's repeated quite a few times in the script.

Reply:
Good idea, done.